centreon/packages/ui/src/Listing/index.tsx

```diff
@@ -494,150 +498,161 @@ const Listing = <TRow extends { id: RowId }>({
             onSelectColumns={onSelectColumns}
           />
         </div>
-        <ParentSize
-          parentSizeStyles={{
-            height: '100%',
-            overflowY: 'auto',
-            width: '100%'
-          }}
-        >
-          {({ height }) => (
-            <Box
-              className={classes.tableWrapper}
-              component="div"
-              style={{
-                height: innerScrollDisabled ? '100%' : `calc(${height}px - 4px)`
-              }}
-            >
-              <Table
-                stickyHeader
-                className={classes.table}
+        {displayCustomListing ? (
+          customListingComponent
+        ) : (
+          <ParentSize
+            parentSizeStyles={{
+              height: '100%',
+              overflowY: 'auto',
+              width: '100%'
+            }}
+          >
+            {({ height }) => (
+              <Box
+                className={classes.tableWrapper}
```

Comment:
you better display the custom listing inside the ParentSize and the Box in order to handle the scroll

Reply:
By the way, for performance purpose, this condition have to be triggered before in order to prevent unused computation and add a ternary condition on the `columnConfiguration` prop